src/vs/vscode.proposed.d.ts

```diff
@@ -1181,6 +1181,45 @@ declare module 'vscode' {
 	}
 	//#endregion
 
+	/**
+	 * Class used to execute an extension callback as a task.
+	 */
+	export class CustomTaskExecution {
```

Comment:
To more closely match the other execution types (ProcessExecution, ShellExecution), this should be CustomExecution.